<launch>
  <arg name="yamlFile" default="$(find abb_driver)/config/robot_cfg.yaml" />
  <rosparam command="load" file="$(arg yamlFile)" />

  <group ns="setup1">
    <!--param name="robot_description"
      command="$(find xacro)/xacro.py '$(find robots_description)/xacro/YumiSingleArm/single_yumi.xacro' arms_interface:=PositionJointInterface grippers_interface:=EffortJointInterface yumi_setup:=default" /-->

    <!-- Load the URDF pkg -->
    <include file="$(find robots_description)/launch/display.launch">
      <arg name="robot" value="yumi" />
    </include>

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

    <node pkg="abb_driver" name="abb_interface" type="abb_interface_node" output="screen" />
    <node pkg="abb_driver" name="abb_interface_gripper" type="abb_interface_gripper_node"
      output="screen" />
    <node pkg="abb_driver" name="controller_bridge" type="controller_bridge_node" output="screen" />
    <node pkg="abb_controllers" name="OneTaskInvKin" type="OneTaskInvKin_node" output="screen" />
  </group>

</launch>